### DISCUSSION TITLE
git&github

### DISCUSSION THREAD

**Discussion by TesterAchievement** [community/community#144714] [GitHub Education] [answered]
### Hi everyone,

What's git and github?

**Comment by Enrico-github** [comment, marked as answer]
Git is a version control system that allows developers to track changes in their codebase. It is designed to handle everything from small to very large projects with speed and efficiency.

**Comment by Enrico-github** [comment]
GitHub is a platform that uses Git for collaboration and hosting repositories online. It is a web-based service where developers can store and share their code.
